crates/bevy_sprite/src/dynamic_texture_atlas_builder.rs

```diff
@@ -27,24 +27,32 @@ impl DynamicTextureAtlasBuilder {
         }
     }
 
-    /// Add a new texture to [`TextureAtlas`].
-    /// It is user's responsibility to pass in the correct [`TextureAtlas`]
+    /// Add a new texture to `atlas_layout`
+    /// It is user's responsibility to pass in the correct [`TextureAtlasLayout`]
```

Comment:
I'd like to see a little more detail here about what exactly that involves.

Reply:
I don't really know, I never used that feature. I clarified the arguments but I did not work on the atlas builder